src/main/java/Duke.java

```diff
@@ -1,10 +1,30 @@
+import java.util.Scanner;
 public class Duke {
+
+    public static void greet() {
+        System.out.println("________________________________");
+        System.out.println("Hello! I'm ChatTPG");
+        System.out.println("What can I do for you?");
+        System.out.println("________________________________");
+    }
+
+    public static void bye(){
+        System.out.println("________________________________");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("________________________________");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greet();
+        Scanner in = new Scanner(System.in);
+        String input;
+        input = in.nextLine();
+        while ((input.compareTo("bye") != 0)){
+            System.out.println("________________________________");
+            TaskManager.handleCommand(input);
+            System.out.println("________________________________");
+            input = in.nextLine();
+        }
```

Comment:
Perhaps you can consider making the whole input part into a method as well.

Reply:
noted, thanks!